Hopsan-release-notes.txt

```diff
@@ -31,6 +31,62 @@
     Version History
 ******************************
 
+2.11.0 (2019-01-??)
+
+--- Enhancements ---
+
+- HopsanCore
+  - Add quantity Resistance with base unit Ohm instead of "ohm".
+    Also adds common unit scales to HopsanGUI.
+  - Add common math functions to embedded script (numhop), see users guide for details.
+    You can now write expression like "x=cos(y)" or "a=atan2(b,c+2)"
+
+- HopsanGenerator:
+  - Added generator for compiler commands that should work with both GCC and Clang.
+    This is only used for library import compilation right now. It will be extended to FMU exporting later.
+    This should enable custom library importing on MacOS from HopsanGUI (but this has not been tested).
+
+- HopsanGUI
+  - Built-in script editor for HCOM scripts.
+  - Improved auto-completer in HCOM terminal.
+  - Case insensitive auto-completer for filter function in component library widget.
+  - It is now possible to add trace variables from multiple selected ports in the "debugging" widget.
+  - Improved find widget Ctrl+F, it now behaves similarly as in other application.
+  - Units scales are now sorted by scale.
+  - Removing three second real-time plotting window limit.
+  - Made it possible to run real-time animation (and plotting) without needing a previous log data generation.
+  - Add "chds" Change diagram size (and position) HCOM command.
+  - Click anywhere on the workspace to quickly add new components. Write the name in the filter box.
+  - Some parts of the HoLC text editor has been merged into HopsanGUI to enhance editing capabilities.
```

Comment:
Actually all parts of the text editor are merged. Only the "generate code skeleton" is missing.